### PR TITLE
Added smoke and unit tests for Swift SDK

### PR DESCRIPTION
**Pull request by jkaster**
- added `smokeTests.swift` and moved some existing tests into it
- `bin/unit_test swift` runs the Swift unit tests
- `bin/smoke swift` runs only the Swift unit and smoke tests

### CODE REVIEW

File: bin/unit_test [hunk 1]
```diff
@@ -48,7 +48,7 @@ if [[ $tests == *swift* ]]
 then
   cd swift/looker || exit
   echo "TODO: Unit testing Swift SDK ..."
```
Comment (joeldodge79) [inline bin/unit_test:50]:
```suggestion
  echo "Unit testing Swift SDK ..."
```

File: bin/unit_test [hunk 2]
```diff
@@ -48,7 +48,7 @@ if [[ $tests == *swift* ]]
...
-  # swift test
+  swift test --filter "api|authToken|constants|models|transport"
```
Comment (joeldodge79) [inline bin/unit_test:51]: too bad there's not a more general filter that would catch these all

**Review by joeldodge79 — CHANGES_REQUESTED** [review]
tiny bit of cleanup

File: bin/unit_test (hunk 2 above)
Comment (jkaster) [reply bin/unit_test:51]: we can _probably_ move those into a `unit` subfolder in a subsequent PR. I think the pattern match will still work.

**Review by joeldodge79 — APPROVED** [review]
LGTM

**Pull request merged**